internal/supervisor/installation.go

```diff
@@ -637,8 +637,10 @@ func (s *InstallationSupervisor) waitForCreationStable(installation *model.Insta
 }
 
 func (s *InstallationSupervisor) configureInstallationDNS(installation *model.Installation, instanceID string, logger log.FieldLogger) string {
+	isActive := true
 	clusterInstallations, err := s.store.GetClusterInstallations(&model.ClusterInstallationFilter{
 		InstallationID: installation.ID,
+		IsActive:       &isActive,
```

Comment:
Here specifically is where I see us creating the new DNS record, but is the old record ever cleaned up?

Reply:
Good Question , here we r updating the existing DNS record with new load balancer value.